mpi/outer_script.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+## REQUIRES: $MPI_TESTS_DIRECTORY $FTC_DIRECTORY $LCSCHEDCLUSTER
+
+corona_COMPILERS="
+gcc
+clang
+intel-classic
+"
+
+corona_MPIS="
+mvapich2
+"
+
+export TESTS="hello
+abort
+version
+"
+
+MPIS="${LCSCHEDCLUSTER}_MPIS"
+COMPILERS="${LCSCHEDCLUSTER}_COMPILERS"
+
+for mpi in ${!MPIS}; do
+    for compiler in ${!COMPILERS}; do
+        flux batch -N2 -n4 --flags=waitable --output=kvs $MPI_TESTS_DIRECTORY/inner_script.sh $mpi $compiler
+    done
+done
+flux job wait --all
+RC=0
```

Comment:
Actually I apologize, I wasn't reading the script carefully before. `flux job wait --all` will return the highest exit code of all jobs, so you could capture `RC=$?` here and wouldn't need the `|| RC=$?` below. Up to you if you want to make that change or not since I think the current version will work.

Reply:
I like this better (because no "magic number" on line 29). Fixed.